pkg/services/object/search/container.go

```diff
@@ -35,13 +34,15 @@ func (exec *execCtx) executeOnContainer() {
 
 		mProcessedNodes[strKey] = struct{}{}
 
+		lg := exec.log.With(zap.String("public key", netmap.StringifyPublicKey(node)))
```

Comment:
it is almost everywhere `log` in our code, why `lg` now?

Reply:
Or it can be `ln`.

Reply:
> it is almost everywhere log in our code, why lg now?

there are many packages called log incl. std